docs/articles/new-expensify/expensify-card/Add-Expensify-Card-to-Apple-or-Google-Pay.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Add Expensify Card to Apple or Google Pay
+description: Pay with your Expensify Card from your Apple or Google Pay wallet
+---
+<div id="new-expensify" markdown="1">
+
+You can use your Expensify Card for contactless in-person payments by adding it to your digital wallet for Apple Pay (for iOS) or Google Pay (for Android).
+
+{% include selector.html values="mobile" %}
+
+{% include option.html value="mobile" %}
+**Apple Pay**
+
+1. Open the Apple Pay app.
+2. Tap the + button.
+3. Tap **Debit or Credit Card**.
+4. Tap **Continue**.
+5. Follow the steps provided to add your virtual card.
+
```

Comment:
Huh, the space between this list and `Google Pay` isn't showing up. @rushatgabhane have you seen that before? I'm not sure why it's happening.
<img width="733" alt="image" src="https://github.com/Expensify/App/assets/5487802/032bbfbb-c7fd-47a8-9b9b-385f7c8983b4">

Reply:
@dangrous let me see what can fix this